paddle/framework/data_transform_test.cc

```diff
@@ -127,7 +127,19 @@ TEST(DataTransform, Layout) {
   }
 
   Tensor dst = out.Get<Tensor>();
```

Comment:
When the Variable is on the GPU, it should also be tested in this unit test.

Reply:
Thanks for pointing these flaws out! These fix will be done in next PR.